### PR TITLE
Create Project / Edit Project - Loading + Sticky Buttons

### PR DESCRIPTION
**Pull request by valentinludu**
### WHAT

Add loading and text near the publish button for connect apps

### WHY

[Card](https://app.charmverse.io/charmverse/summer-loading-indicator-and-text-when-publishing-project-9283198065757443)

### CODE REVIEW

File: apps/connect/components/projects/components/ProjectForm.tsx [hunk 1]
```diff
@@ -18,155 +18,164 @@ export function ProjectForm({
...
-        dataTest='project-form-farcaster-values'
-        placeholder='https://warpcast.com/charmverse'
-      />
-      <Stack>
-        <FormLabel id='project-twitter'>X</FormLabel>
-        <Stack direction='row' gap={1} alignItems='center'>
-          <Typography color='secondary' width={250}>
-            https://x.com/
-          </Typography>
+    <>
```
Comment (motechFR) [inline apps/connect/components/projects/components/ProjectForm.tsx:21]: Does it make sense to componentise this and share with Sunny?

**Review by motechFR — APPROVED** [review]

File: apps/sunnyawards/app/loading.tsx [hunk 2]
```diff
@@ -1,7 +1,6 @@
+import { LoadingComponent } from '@connect-shared/components/common/Loading/LoadingComponent';
 import Box from '@mui/material/Box';
 
-import { LoadingComponent } from 'components/common/LoadingComponent';
-
 export default function Loading() {
```
[the remark this answers is not in the document]
Comment (valentinludu) [reply apps/sunnyawards/app/loading.tsx:4]: I did that with the LoadingComponent.

Loading.tsx is a page and can't be.

File: apps/connect/components/projects/components/ProjectForm.tsx (hunk 1 above)
Comment (valentinludu) [reply apps/connect/components/projects/components/ProjectForm.tsx:21]: We are going to deviate from Sunny app This one is quite a big one so I thought not to do it. I prefer to share only small components without too much logic.

**Review by motechFR — APPROVED** [review]

File: apps/connect/components/projects/components/ProjectForm.tsx (hunk 1 above)
Comment (motechFR) [reply apps/connect/components/projects/components/ProjectForm.tsx:21]: Agreed

**Pull request merged**